certified-connectors/snowflake/apiProperties.json

```diff
@@ -5,9 +5,11 @@
         "type": "oauthSetting",
         "oAuthSettings": {
           "identityProvider": "aad",
+          "clientId": null,
+          "clientSecret" : null,
           "scopes": [],
-          "redirectMode": "Global",
-          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "redirectMode": "GlobalPerConnector",
+          "redirectUrl": "",

```

Comment:
Hello @naisonslalom, 

Please remove redirectUrl completely, as this is not needed.

Reply:
@vmanoharas, done, please verify